pkg/platform/runtime/target/target.go

```diff
@@ -33,6 +34,13 @@ const (
 	triggerUnknown  Trigger = "unknown"
 )
 
+// usageTriggers are triggers that indicate actual usage of the runtime (as oppose to simply making changes to the runtime)
+var usageTriggers = []Trigger{TriggerActivate, TriggerScript, TriggerDeploy, TriggerExec}
```

Comment:
If `TriggerDeploy` is a usage trigger should `TriggerDefault` be as well? I could also argue to include default as it's similar to activate

Reply:
I didn't use it because looking at the code it doesn't seem to be about default at all. Though I should've renamed it while I was at it. Will do that now.